src/cargo/ops/fix.rs

```diff
@@ -222,12 +208,17 @@ pub fn fix_maybe_exec_rustc() -> CargoResult<bool> {
 
     let args = FixArgs::get();
     trace!("cargo-fix as rustc got file {:?}", args.file);
+
     let rustc = args.rustc.as_ref().expect("fix wrapper rustc was not set");
+    let workspace_rustc = std::env::var("WORKSPACE_RUSTC_WRAPPER")
```

Comment:
I am worried that this is essentially the same as using `is_primary`, probably need some check here to conditionally wrap only when the current crate is a workspace member.

Reply:
I think it is OK for now.  Non-workspace packages have cap-lints applied which prevents `cargo fix` from doing anything. `cargo fix` should probably just reject `-p` for a non-member to avoid confusion, but isn't too important.